test/unit/index.js

```diff
@@ -175,64 +147,205 @@ describe('minimal-cipher', function() {
 
       it('should encrypt a simple Uint8Array', async function() {
         const data = getRandomUint8();
-        const result = await cipher.encrypt({data, recipients, keyResolver});
+        const result = await cipher.encrypt({
+          data,
+          recipients: recipient,
+          keyResolver
+        });
         result.should.be.a.JWE;
+        result.recipients.length.should.eql(1);
```

Comment:
Here and throughout.

```suggestion
        result.recipients.length.should.equal(1);
```

Reply:
addressed here: https://github.com/digitalbazaar/minimal-cipher/pull/31/commits/23a6e808dcafc6e080ac48aa7d584fa23f0aca45